lib/puppet/provider/package/portage.rb

```diff
@@ -121,7 +121,7 @@ def qatom
       else
         search = '=' + @resource[:name] + '-' + "#{should}"
       end
-      search_output = qatom_bin(*([search, '--format', output_format]))
```

Comment:
Are these asterisks important to this provider?

Reply:
I believe the asterisk was removed as a part of the [Lint/RedundantSplatExpansion](https://msp-greg.github.io/rubocop/RuboCop/Cop/Lint/RedundantSplatExpansion.html) cop. With the extra parenthesis removed as a part of the Style/RedundantParenthesis cop, the line looks like:
`search_output = qatom_bin(*[search, '--format', output_format])`

Rubocop thinks the splat operator isn't needed in this case and each of those params can be passed in without it but tbh I'm not too familiar with the splat operator and I'm okay undoing that change if we're unsure.